src/features/animation.ts

```diff
@@ -16,7 +16,7 @@
 import {property} from 'lit-element';
 
 import ModelViewerElementBase, {$needsRender, $onModelLoad, $scene, $tick, $updateSource} from '../model-viewer-base.js';
-import {Constructor} from '../utils.js';
+import {Constructor} from '../utils';
```

Comment:
I made a `utils` folder because it felt like too big a change to add [Timer](https://github.com/GoogleWebComponents/model-viewer/pull/469/files#diff-358694c697d2ba87ef835140c3e1e7a7) to the `utils.js` file. `utils.js` is not `utils/index.js`.

Reply:
Ok cool. Was wondering about that cause saw `rollup-plugin-node-resolve` being used which well does node style resolution 😀

Would you simply prefer that all imports are something like `../utilities/index.js` or `../utilities/utilities.js` or?

It might be nice to split up all functionality in `utilities/index.js` to separate files but 🤷‍♂️. Def not on this PR.

Reply:
You could have `utilities.js` and `utilities/timer.js`? We have a precedent for that: `features/loading.js` and `features/loading/status-announcer.js`.